src/pages/media.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Helmet from "react-helmet";
+import Layout from "../layout";
+import Media from "../components/Media/Media";
+import config from "../../data/SiteConfig";
+import Img from "gatsby-image"
+import {useStaticQuery, graphql} from "gatsby"
+
+const Screenshots = () => {
+  const data = useStaticQuery( graphql`
+  query Images{
+    images: allFile(filter: {relativeDirectory: { eq: "images"} }){
+      nodes {
+        id
+        childImageSharp {
+          fixed(width: 200, height: 200) {
+            ...GatsbyImageSharpFixed
+          }
+      }
+    }
+  }
+  }
+`
+  )
+  console.log(data);
+  return (
+  <Layout>
+    <div className="media-container">
+      <Helmet title={`Media | ${config.siteTitle}`} />
+      <Media />
+    </div>
+    <div className="screenshots">
```

Comment:
Great update!
Although can you move this whole thing inside the `Media` component and then rename it to gallery? Will also fix the gallery component issue that is present. Also adding an a popup screen that would display the full resolution image when clicked on the thumbnail. You can make the gallery universal if you are passing some parameter or prop to the Gallery component. In this way we could use the gallery to display images in blog posts, module pages etc.

**_Why ?_**
Try to keep all the logical data into components until necessary to be implemented into the page. Keep the pages to interact with the components along with static data.